psi4/src/psi4/libmints/vector.cc

```diff
@@ -265,10 +265,6 @@ void Vector::add(const SharedVector &other) { axpy(1.0, *other.get()); }
 
 void Vector::add(const Vector &other) { axpy(1.0, other); }
 
-void Vector::subtract(const SharedVector &other) { axpy(-1.0, *other.get()); }
-
-void Vector::subtract(const Vector &other) { axpy(-1.0, other); }
-
```

Comment:
I don't remember the whole story, but but I'm willing to give up subtract so long as it doesn't break v2rdm. https://github.com/edeprince3/v2rdm_casscf/blob/4375855616ca440b093b5cde20e45542ece6681d/v2rdm_solver.cc#L1761

Reply:
Mistake on my part, thanks for catching it!